Watership_Down_Exercise/Hill.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Watership_Down_Exercise.Enum;
+
+namespace Watership_Down_Exercise
+{
+    /// <summary>
+    /// The Hill class is used to simulate a hill that contains bunnies on it.
+    /// </summary>
+    class Hill
+    {
+        private List<Bunny> bunniesList;
```

Comment:
If it's a private variable and has no access outside its class it's called a data members and should start with _ (_bunnies)